repos/spack_repo/builtin/packages/civetweb/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+
+from spack.package import *
+
+
+class Civetweb(CMakePackage):
+    """CivetWeb is an easy to use, powerful, C (C/C++) embeddable
+    web server with optional CGI, SSL and Lua support."""
+
+    homepage = "https://github.com/civetweb/civetweb"
+    url = "https://github.com/civetweb/civetweb/archive/refs/tags/v1.16.tar.gz"
+
+    maintainers("wdconinc")
+
+    license("MIT", checked_by="wdconinc")
+
+    version("1.16", sha256="f0e471c1bf4e7804a6cfb41ea9d13e7d623b2bcc7bc1e2a4dd54951a24d60285")
```

Comment:
```suggestion
    version("1.17", commit="5864b55a94f4b5238155cbf2baec707f0fa2ba6d")
    version("1.16", sha256="f0e471c1bf4e7804a6cfb41ea9d13e7d623b2bcc7bc1e2a4dd54951a24d60285")
```

Mainly choosing 1.17 here because the vendored version from ROOT also does it.

Reply:
Why is this 1.17? It hasn't been released yet so we shouldn't run ahead of the project. How about `1.16-213-g5864b55a`?

Reply:
```suggestion
    version("1.16-213-g5864b55a", commit="5864b55a94f4b5238155cbf2baec707f0fa2ba6d")
    version("1.16", sha256="f0e471c1bf4e7804a6cfb41ea9d13e7d623b2bcc7bc1e2a4dd54951a24d60285")
```

Reply:
(Mainly for completeness at this point) I have chosen 1.17, mainly because that was also the value to which the vendored config was changed: https://github.com/root-project/root/blob/ef30a48d2c34d27d4e301457f59479a9edce9c4a/net/http/civetweb/civetweb.h#L26-L29

Reply:
I think that's just the header versioning, which probably has claimed 1.17 since the first commit after 1.16 was released.